modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc

```diff
@@ -85,7 +83,7 @@ image::api-manager-status-for-service-mesh.png[85%, 85%]
 
 [[bind_using_ASM]]
 
-== Bind an API Created Using Anypoint Service Mesh
+== Bind an API created using Anypoint Service Mesh API name.
```

Comment:
Should "API name" be at the end of this heading?
Why is there a period at the end?

Reply:
Done! Thanks!